From: Andrzej Pietrasiewicz <andrzej.p@collabora.com>
Date: Fri, 3 Jul 2020 12:43:52 +0200
Subject: acpi: thermal: Don't call thermal_zone_device_is_enabled()
Patch-mainline: v5.9-rc1
Git-commit: 7a7d66f26b0ff47e7edbc9df884ad711ed918992
References: jsc#SLE-16407

thermal_zone_device_update() can now handle disabled thermal zones, so
the check here is not needed.

Signed-off-by: Andrzej Pietrasiewicz <andrzej.p@collabora.com>
Signed-off-by: Daniel Lezcano <daniel.lezcano@linaro.org>
Link: https://lore.kernel.org/r/20200703104354.19657-2-andrzej.p@collabora.com
Acked-by: Lee, Chun-Yi <jlee@suse.com>
---
 drivers/acpi/thermal.c |    3 ---
 1 file changed, 3 deletions(-)

--- a/drivers/acpi/thermal.c
+++ b/drivers/acpi/thermal.c
@@ -499,9 +499,6 @@ static void acpi_thermal_check(void *dat
 {
 	struct acpi_thermal *tz = data;
 
-	if (!thermal_zone_device_is_enabled(tz->thermal_zone))
-		return;
-
 	thermal_zone_device_update(tz->thermal_zone,
 				   THERMAL_EVENT_UNSPECIFIED);
 }
